ci_config.json

```diff
@@ -43,6 +43,18 @@
       "boost"
     ]
   },
+  "azure-sdk-for-cpp": {
+    "build_on": {
+      "msys2": false,
+      "windows": false
```

Comment:
There seems to be some kind of issue with missing symbols on Windows. I can't tell if it has to do with the export macros or maybe even some missing Windows libraries that are needed.

For now I've just punted on it, in hopes that someone with more Windows knowledge and interest can fix this at a later date